src/microbe_stage/BiomeConditions.cs

```diff
@@ -199,7 +199,7 @@ public bool TryGetCompound(Compound compound, CompoundAmountType amountType, out
     ///   Specifies the compound cloud spawn sizes for all new non-environmental compounds
     /// </param>
     public void ApplyLongTermCompoundChanges(Biome biomeDetails, Dictionary<Compound, float> changes,
-        IReadOnlyDictionary<Compound, float> newCloudSizes)
+        IReadOnlyDictionary<Compound, float>? newCloudSizes)
```

Comment:
I think this shouldn't be made nullable because it signifies a major problem if an existing cloud size is not known. So I wouldn't want to make it easier to make bugs happen by making this a nullable parameter.